platform/viewer/src/appInit.js

```diff
@@ -60,6 +61,7 @@ async function appInit(appConfigOrFunc, defaultExtensions, defaultModes) {
     CineService.REGISTRATION,
     UserAuthenticationService.REGISTRATION,
     PanelService.REGISTRATION,
+    [StateSyncService.REGISTRATION, appConfig.stateSyncService],
```

Comment:
What configuration is there for the state sync service? Am I missing something?

Reply:
Nothing right now, but the general pattern should be that IF one wants to define configuration in the appConfig, that is the way to do it. If you prefer not to do it that way given that it isn't used, I can remove it.